.github/workflows/test-linux.yml

```diff
@@ -3,13 +3,10 @@ name: test-linux
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore: ['**.md', doc/**, AUTHORS, .mailmap, .github/**, '!.github/workflows/test-linux.yml']
   push:
-    branches:
-      - master
-      - main
-      - canary
-      - v[0-9]+.x-staging
-      - v[0-9]+.x
+    branches: [master, main, canary, 'v[0-9]+.x-staging', 'v[0-9]+.x']
+    paths-ignore: ['**.md', doc/**, AUTHORS, .mailmap, .github/**, '!.github/workflows/test-linux.yml']
```

Comment:
We want to run at least one build for doc changes -- at least one test checks for consistency between documentation and run time (hence needing a built `node` binary) behaviour: 
- [test/parallel/test-process-env-allowed-flags-are-documented.js](https://github.com/nodejs/node/blob/master/test/parallel/test-process-env-allowed-flags-are-documented.js)

Reply:
I think there are also addon tests that are built from the docs?

Reply:
```diff
-     paths-ignore: ['**.md', doc/**, AUTHORS, .mailmap, .github/**, '!.github/workflows/test-linux.yml']
+     paths-ignore: ['AUTHORS, .mailmap, .github/**, '!.github/workflows/test-linux.yml']
```
?